easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,133 @@ def extra_options():
+
+            # optionally enable support for Python as alternative interpreter
+            python_root = get_software_root('Python')
+            if python_root:
+                self.with_python = True
+                self.cfg.update('configopts', "--with-nrnpython=%s/bin/python" % python_root)
+
+            # determine host CPU type
+            cmd = "./config.guess"
+            (out, ec) = run_cmd(cmd, simple=False)
 
-        # enable support for distributed simulations if desired
-        if self.cfg['paranrn']:
-            self.cfg.update('configopts', '--with-paranrn')
+            self.hostcpu = out.split('\n')[0].split('-')[0]
+            self.log.debug("Determined host CPU type as %s" % self.hostcpu)
 
-        # specify path to InterViews if it is available as a dependency
-        interviews_root = get_software_root('InterViews')
-        if interviews_root:
-            self.cfg.update('configopts', "--with-iv=%s" % interviews_root)
+            # determine Python lib dir
+            self.pylibdir = det_pylibdir()
+
+            # complete configuration with configure_method of parent
+            ConfigureMake.configure_step(self)
         else:
-            self.cfg.update('configopts', "--without-iv")
+            # enable support for distributed simulations if desired
+            if self.cfg['paranrn']:
+                self.cfg.update('configopts', '-DNRN_ENABLE_MPI=ON')
+            else:
+                self.cfg.update('configopts', '-DNRN_ENABLE_MPI=OFF')
 
-        # optionally enable support for Python as alternative interpreter
-        python_root = get_software_root('Python')
-        if python_root:
-            self.with_python = True
-            self.cfg.update('configopts', "--with-nrnpython=%s/bin/python" % python_root)
+            # specify path to InterViews if it is available as a dependency
+            interviews_root = get_software_root('InterViews')
+            if interviews_root:
+                self.cfg.update('configopts', "-DIV_DIR=%s" % interviews_root)
+            else:
+                self.cfg.update('configopts', "-DNRN_ENABLE_INTERVIEWS=OFF")
 
-        # determine host CPU type
-        cmd = "./config.guess"
-        (out, ec) = run_cmd(cmd, simple=False)
+            # no longer used it seems
+            self.hostcpu = ''
 
-        self.hostcpu = out.split('\n')[0].split('-')[0]
-        self.log.debug("Determined host CPU type as %s" % self.hostcpu)
+            # optionally enable support for Python as alternative interpreter
+            python_root = get_software_root('Python')
+            if python_root:
+                self.with_python = True
+                self.cfg.update('configopts', "-DNRN_ENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=%s/bin/python" % python_root)
+                self.cfg.update('configopts', "-DNRN_ENABLE_MODULE_INSTALL=ON "
+                                "-DNRN_MODULE_INSTALL_OPTIONS='--prefix=%s'" % self.installdir)
```

Comment:
typically these options are not required (I rarely provide these explicitly).

Reply:
From what I could see, they are required to get the python module installed under `lib/pythonX.Y/site-packages`

Reply:
Understood! this is fine then!